src/Microsoft.ML.Data/Transforms/ConvertTransform.cs

```diff
@@ -169,6 +179,25 @@ private static VersionInfo GetVersionInfo()
         // This is parallel to Infos.
         private readonly ColInfoEx[] _exes;
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="name">Name of the output column.</param>
+        /// <param name="source">Name of the column to be converted.  If this is null '<paramref name="name"/>' will be used.</param>
+        /// <param name="resultType">The expected type of the converted column.</param>
+        /// <param name="keyRange">For a key column, this defines the range of values.</param>
+        public ConvertTransform(IHostEnvironment env,
+            IDataView input,
+            string name,
+            string source = null,
+            DataKind? resultType = null,
+            KeyRange keyRange = null)
```

Comment:
>KeyRange keyRange = null [](start = 12, length = 24)

I don't think conversions among key ranges are common enough that they are worthy of a convenience constructor.